### PR TITLE
[SSL for SaaS] File organization

### PR DESCRIPTION
**Pull request by mmalden**

### CODE REVIEW

File: content/ssl/ssl-for-saas/related-products/_index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+pcx-content-type: overview
+title: Related Products
+weight: 3
+---
+
+# Related Products
+
+Cloudflare for SaaS allows you to pair Cloudflare products with your hostname management for increased security, performance, and reliability.
+* [WAF for SaaS](/ssl/ssl-for-saas/related-products/waf-for-saas) allows you to assign specific security rules to each hostname using Cloudflare's [Web Application Firewall (WAF)](/waf/).
```
Comment (kodster28) [inline content/ssl/ssl-for-saas/related-products/_index.md]: If you're only listing examples contained in the subfolder, might be worth just using a `{{<directory-listing>}}` component to do that dynamically.

**Review by Princeobinna — COMMENTED** [review]
Approve assets

**Review by kodster28 — APPROVED** [review]
Approved from PCX review. Will wait to push live for later this afternoon.

File: content/ssl/ssl-for-saas/related-products/_index.md (hunk 1 above)
Comment (Princeobinna) [reply content/ssl/ssl-for-saas/related-products/_index.md]: Done approve

Comment (Princeobinna) [reply content/ssl/ssl-for-saas/related-products/_index.md]: And how I'm I doing that ????

**Comment by Princeobinna** [comment]
Approve assets

**Comment by kodster28** [comment]
Also addresses #5079.

**Pull request merged**